democracy/views/section.py

```diff
@@ -52,3 +54,34 @@ def get_queryset(self):
         if not hearing.closed:
             queryset = queryset.exclude(type__identifier=InitialSectionType.CLOSURE_INFO)
         return queryset
+
+
+class RootSectionImageSerializer(SectionImageSerializer):
+    hearing = serializers.SerializerMethodField()
+
+    class Meta(SectionImageSerializer.Meta):
+        fields = SectionImageSerializer.Meta.fields + ['section', 'hearing']
+
+    def get_hearing(self, section_image):
+        return section_image.section.hearing.id
```

Comment:
Yigh, at least use `hearing_id` (and it would be a darn good idea to ensure `.section` is either cached in or `.select_related()`ed.